dashboard/config/routes.rb

```diff
@@ -77,6 +77,9 @@ module OPS
         collection do
           post 'bulk_add'
         end
+        member do
+          post 'remove'
```

Comment:
wasn't sure if I should make this a `DELETE` since the other API was, but we're not actually deleting the user, just removing them from the section, so I felt like `POST` might be more appropriate. could be convinced otherwise.